src/Native/System.Net.Http.Native/pal_versioninfo.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "pal_versioninfo.h"
+
+#include <curl/curl.h>
+
+extern "C" int32_t GetCurlVersionInfo(int32_t* age, int32_t* supportsSsl, int32_t* supportsAutoDecompression)
+{
+    curl_version_info_data* versionInfo = curl_version_info(CURLVERSION_NOW);
+
+    if (!versionInfo || !age || !supportsSsl || !supportsAutoDecompression)
```

Comment:
It looks like we call this from one place and we pass in each variable...why not make these asserts that they are non-nullptr and then just set them?